.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: enhancement
+assignees: ''
+
+---
+
+**Do you believe that the feature you want to have implement makes any sense?**
```

Comment:
this should probably be made more sensical, like "why is this feature needed?" fitting with the line that immediately follows it

Reply:
This target a commit that should not be here

Reply:
@0ddlyoko what do you mean?

Reply:
ohhh true. Means this change shouldn't be in this commit/PR since it's not related to the join message wording, unless we want to update the PR to "change wording and edit issue templates"

Reply:
https://prnt.sc/uzhg85
This commit